test/post.test.js

```diff
@@ -64,7 +71,16 @@ test.serial('Post run with github token', async t => {
   // Verify the published release note
   t.deepEqual(result, {
     published: true,
-    release: {owner, repo, tag_name: tagName, name: tagName, target_commitish: branch, draft: debug, body: releaseLog},
+    release: {
+      owner,
+      repo,
+      tag_name: tagName,
+      name: tagName,
+      target_commitish: branch,
+      draft: debug,
+      body: releaseLog,
+      prerelease: expectPreRelease,
```

Comment:
Can you rename `expectPreRelease` into `prerelease` ?
It's an expected value [there](https://github.com/semantic-release/semantic-release/pull/457/files#diff-0febbd8a0bc438ee56826534d7358b1cR57) but it's the "obtain" value here, in the sense it's what the github API client sent. So better to call it just `prerelease`.